core/store/database.go

```diff
@@ -0,0 +1,22 @@
+package store
+
+// Database provides access to the underlying data base for get set and delete operations of nonconsensus Data
+type DatabaseService interface {
+	GetDataBase() NonConsensusStore
+}
+
+// NonConsensusStore is a simple key-value store that is used to store non-consensus data.
+// Note the non-consensus data is not committed to the blockchain and does not allow iteration
+type NonConsensusStore interface {
```

Comment:
Asking to learn: How will this be exposed/provided to modules? Through `Environment`?

Reply:
yes through environment